### DISCUSSION TITLE
Traditional saves and save state locations?

### DISCUSSION THREAD

**Discussion by RivermanPaul** [eduardofilo/RG350_adam_image#158] [Q&A] [answered]
Sorry if this is a dumb question, but where are save states and traditional saves stored when using the Adam image?  I can't find this information anywhere.  The reason I ask is that I know I'll want another retro console in the future (that may or may not support the Adam image), and I'd like to be able to transfer some of my saves/save states, even if it's a manual process.  Thank you!

**Comment by RivermanPaul** [comment, marked as answer]
I figured it out. For anyone reading this: The save files aren't visible normally.  You need to use the Py Backup app to export the save files (see https://youtu.be/5pfYRm34hs0?t=228).  Once you do this, the backups be visible in "backups/" on your games SD card.  You can then unzip the relevant folder (such as RetroArch saves for most of the Nintendo stuff, or PCSX4All for PS1) then drill down until you find the save states and saved games, then you can store and use them to your computer or other device in whatever way is most convenient.  To restore them to this (or another compatible) device, I would GUESS the easiest way is to do a backup export like this so that you can see the file structure, unzip the relevant folders, paste your saves in, zip it again, then use Py Backup to restore them.

**Reply by eduardofilo** [reply]
You have the details of the paths saved in each entry in Py Backup's config file:

https://github.com/eduardofilo/RG350_adam_image/blob/master/manual_build/config.ini
